lib/storage.js

```diff
@@ -164,6 +164,41 @@ exports.has = function(key, callback) {
 };
 
 /**
+ * @summary Get the list of saved keys
+ * @function
+ * @public
+ *
+ * @fulfil {String[]} - saved keys
+ * @returns {Promise}
+ *
+ * @example
+ * const storage = require('electron-json-storage');
+ *
+ * storage.keys().then(function(keys) {
+ *   for (var key of keys) {
+ *     console.log('There is a key called: ' + key);
+ *   }
+ * });
+ *
+ * @example
+ * const storage = require('electron-json-storage');
+ *
+ * storage.keys(function(error, keys) {
+ *   if (error) throw error;
+ *
+ *   for (var key of keys) {
+ *     console.log('There is a key called: ' + key);
+ *   }
+ * });
+ */
+exports.keys = function(callback) {
+  const userData = utils.getUserDataPath();
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).